fmp_data/lc/__init__.py

```
@@ -82,80 +72,80 @@
     return fmp_key, openai_key
 
 
-def setup_registry(client: FMPDataClient) -> EndpointRegistry:
+def setup_registry(client: "FMPDataClient") -> Any:  # EndpointRegistry
     """Initialize and populate endpoint registry."""
-    registry = EndpointRegistry()
+    if not is_langchain_available():
+        raise ImportError(
+            "LangChain dependencies required. Install with: pip install 'fmp-data[langchain]'"
```

Ruff (0.11.9)

Line too long (94 > 88)

(E501)